app/Http/Controllers/MembersController.php

```diff
@@ -39,13 +46,13 @@ public function index()
 	}
 
 	# Member details page
-	public function show(Member $member)
+	public function show(Member $member, string $viewType = 'admin')
```

Comment:
I still don't like this "viewType" distinction. What this does it distinguishes between 2 different views. So putting this in one view end up with a lot of conditionals in the view.
I rather see having two views: one for profile one for admin, which resuses / includes allot of common components between the two.
But maybe addressing this can be better done as a follow-up